docs/core/app-types.md

```diff
@@ -34,8 +34,7 @@ There are two main types that we can observe:
 
 ## Portable applications
```

Comment:
What would be a better term to describe them? "Portable" here is at least descriptive -- the apps are portable from one .NET Core installation to another, given that they depend on the external runtime and class libraries. I can't think of a better term. Should we open an issue?